charts/rancher-istio/1.1.5/values.yaml

```diff
@@ -250,7 +280,9 @@ global:
 
   proxy_init:
     # Base name for the proxy_init container, used to configure iptables.
-    image: proxy_init
+    # TODO update to rancher/istio-proxy_init when the image is pushed into rancher repo
```

Comment:
Put the TODO to the PR description. Do we mirror chart images manually at the moment? I wonder if we can traverse system-chart values.yaml and extract images according to the keys in our mirror scripts.